app/views/contexts/summary/group.py

```diff
@@ -56,19 +62,19 @@ def __init__(
     def _build_blocks_and_links(
         self,
         *,
-        group_schema,
-        routing_path,
-        answer_store,
-        list_store,
-        metadata,
-        response_metadata,
-        schema,
-        location,
-        return_to,
-        progress_store,
-        language,
-        return_to_block_id,
-    ):
+        group_schema: dict[str, list[Any]],
```

Comment:
Should be `Mapping` unless we need `dict` specific methods.

Reply:
👍